docs/tools/vdb_table/data/cratedb.json

```diff
@@ -3,9 +3,9 @@
   "links": {
     "docs": "https://cratedb.com/solutions/vector-database",
     "github": "https://github.com/crate/crate",
-    "website": "http://www.cratedb.com/",
+    "website": "https://cratedb.com/",
     "vendor_discussion": "https://github.com/superlinked/VectorHub/discussions/98",
-    "poc_github": "https://github.com/marijaselakovic",
+    "poc_github": "https://github.com/crate/cratedb-examples",
```

Comment:
this is meant to be the github handle for the point of contact for the DB. Ideally someone who will keep this data updated.

Reply:
@BaurzhanSakhariev could you revert this change, or provide an appropriate point of contact's github user handle here?